src/commands/api/getDatabase.ts

```diff
@@ -7,6 +7,6 @@ import { DocDBDatabaseTreeItem } from '../../docdb/tree/DocDBDatabaseTreeItem';
 import { ext } from '../../extensionVariables';
 import { MongoDatabaseTreeItem } from '../../mongo/tree/MongoDatabaseTreeItem';
 
-export async function getDatabase(): Promise<String> {
+export async function getDatabase(): Promise<string> {
     return (await ext.tree.showTreeItemPicker([MongoDatabaseTreeItem.contextValue, DocDBDatabaseTreeItem.contextValue])).fullId;
```

Comment:
Also, we should not be returning the raw id.  As mentioned in the meeting, the ID format that's exposed externally from cosmos has to remain unchanged, even if the internal implementation changes.

So you need a conversion function to/from fullId to the external Id, and we should discuss what the external ID looks like.